.github/workflows/linux_test.yml

```diff
@@ -0,0 +1,48 @@
+name: "Linux test"
```

Comment:
We want to test on cross platform - windows, linux, macOS.

So, name/file name to be `ci`.